K8s-deployment/Charts/consent-validator/example-secrets/secrets/config.json

```
@@ -0,0 +1,46 @@
+{
+    "version": "1.0",
+    "postgresOptions": {
+      "databaseIP": "database.iudx.io",
+      "databasePort": "5432",
+      "databaseName": "adex_consent_manager",
+      "databaseSchema": "adexconsent",
+      "databaseUserName": "adex_consent_user",
+      "databasePassword": "<Postgres Password of adex_consent_user >",
```

Replace with the shell command to retrieve the password. 
```suggestion
      "databasePassword": "<psql-pass:- cat iudx-deployment/K8s-deployment/Charts/postgresql/secrets/passwords/postgres-consent-password>",
```